cves/CVE-2012-0858.yml

```diff
@@ -174,10 +173,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: Similar vulnerabilites were found by Mateusz Jurczyk and Gynvael Coldwind. This was done by Michael Niedermayer and Kurt Seifried of Red Hat Security
```

Comment:
Does it say how the vulnerability was found?
Also same 80 character spacing recommendation from above.